packages/cli/src/commands/validatorset.ts

```diff
@@ -0,0 +1,21 @@
+import { BaseCommand } from '../base'
+
+export default class ValidatorSet extends BaseCommand {
+  static description = 'Outputs the current validator set'
+
+  static flags = {
+    ...BaseCommand.flags,
+  }
+
+  static examples = ['validatorset']
+
+  async run() {
+    const validators = await this.kit.contracts.getValidators()
```

Comment:
I think this is a great case for having `getValidatorsSet()` in the Validators Wrapper.

having a `getLength` + `getElementWithIdx()` sound too loo level for the wrapper